project_event/static/description/index.html

```diff
@@ -0,0 +1,128 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <h2 class="oe_slogan" style="color:#875A7B;">Project Event</h2>
+        <h3 class="oe_slogan">Project Event is an extension to Project resource calendar bring more features for event management</h3>
+        <h4 class="oe_slogan">Used by Université de Montréal, specially built for Music Acadamy management, Theater management for Shows, Events and Spectacles.</h4>
+        <div class="oe_demo oe_picture oe_screenshot">
+            <img src="icon.png">
+        </div>
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span6">
+            <h2 class="oe_slogan" style="color:#875A7B; text-align: left;">Create Events, Activites and Tasks booking resources</h2>
+            <h3 class="oe_slogan" style="text-align: left;">Time saving features</h3>
+            <p class="oe_mt32">
+                The project module was augmented with fields and functionnalities allowing an Event organiser structure and plan different activities and tasks related to a specific event
+            </p>
+            <ul>
+                <li>Top down effect : When changing information in the Top event element, related acivities and tasks are affected. Example: The name of the Client</li>
+            </ul>
+        </div>
+        <div class="oe_span6">
+            <img class="oe_picture oe_screenshot" src="main_screenshot.png"/>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span6">
+            <h2 class="oe_slogan" style="color:#875A7B; text-align: left;">Setup categories with different colors</h2>
+            <p class="oe_mt32">
+                Easy setup categories with different colors.
+            </p>
+        </div>
+        <div class="oe_span6">
+            <img class="oe_picture oe_screenshot" src="setup_categories.png"/>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span6">
+            <h2 class="oe_slogan" style="color:#875A7B; text-align: left;">Status management</h2>
+            <p class="oe_mt32">
+                The status are used to followup on where the Event the Activity and the Task are in the process of execution.<br/>
+                When changing a task (Only tasks) status from draft to option a clone is created in the Project Resource Calendar Module.
```

Comment:
I would not use the word "clone". 

```suggestion
                When changing a task  and changing its status from draft to option, a calendar event(meeting with reservations) is created.
```